ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -27,14 +27,23 @@ public final class Atomic<Value> {
 	/// Initializes the variable with the given initial value.
 	public init(_ value: Value) {
 		_value = value
+		let result = pthread_mutex_init(&mutex, nil)
+		assert(0 == result, "Failed to initialize mutex with error \(result).")
```

Comment:
This can just be `assert(result == 0, ...)`. No need for the defensive ordering since `Swift` will warn you if you accidentally make it an assignment :)